src/ui/components/Button/styles.scss

```diff
@@ -66,6 +66,7 @@ $puffy-font-size: 15px;
         font-size: $micro-font-size;
         height: auto;
         min-height: 24px;
+        padding: 1px 16px;
```

Comment:
This will affect all buttons so it's not the right fix.

The problematic button is already special:

https://github.com/mozilla/addons-frontend/blob/9fd03080ce7b0d2d4db3ab7e748e7fa59f08d422/src/amo/components/RatingManager/styles.scss#L12